rafx-api/src/api.rs

```diff
@@ -129,6 +152,8 @@ impl RafxApi {
             RafxApi::Vk(inner) => inner.destroy(),
             #[cfg(feature = "rafx-metal")]
             RafxApi::Metal(inner) => inner.destroy(),
+            #[cfg(feature = "rafx-gles2")]
+            RafxApi::Gles2(inner) => inner.destroy(),
             #[cfg(any(
                 feature = "rafx-empty",
                 not(any(feature = "rafx-metal", feature = "rafx-vulkan"))
```

Comment:
Find / replace all instances of

`not(any(feature = "rafx-metal", feature = "rafx-vulkan"))` with `not(any(feature = "rafx-metal", feature = "rafx-vulkan", feature="rafx-gles2"))`?